src/main/java/qna/domain/Answer.java

```diff
@@ -99,12 +92,28 @@ public void setDeleted(boolean deleted) {
         this.deleted = deleted;
     }
 
+    public Question getQuestion() {
+        return question;
+    }
+
+    public void setQuestion(Question question) {
+        this.question = question;
+    }
+
+    public User getUser() {
+        return user;
+    }
+
+    public void setUser(User user) {
+        this.user = user;
+    }
+
     @Override
     public String toString() {
         return "Answer{" +
                 "id=" + id +
-                ", writerId=" + writerId +
-                ", questionId=" + questionId +
+                ", writerId=" + user +
+                ", questionId=" + question +
```

Comment:
```suggestion
                ", writerId=" + user.getId() +
                ", questionId=" + question.getId() +
```
참고로 양방향 참조관계에서 객체를 그대로 출력하게되면
서로가 서로의 `toString` 메서드를 호출하게 되어 스택오버플로우가 발생하게됩니다.

Reply:
앗.. 영한님 강의 들으면서 명심해야지 라고 생각했던 부분을 바로 실수했네요.
리뷰 감사드립니다 :+1